magic/core/src/main/java/link/magic/android/core/relayer/WebViewWrapper.kt

```diff
@@ -182,4 +206,27 @@ class WebViewWrapper internal constructor(context: Context, private val urlBuild
             }
         }
     }
+
+    private fun maybeRebuildWebView(context: Context) {
+         if (messageSentId != null && messageReceivedId != null && messageSentId!! != messageReceivedId!!) {
```

Comment:
Can we use `messageReceiptId` instead of `messageReceivedId`? Received is to general and might not be clear